src/main/java/org/mifos/integrationtest/config/AmsConnectorConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.mifos.integrationtest.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.*;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class AmsConnectorConfig {
+
+    @Value("${mifos-connector.endpoints.account_status}")
+    public String accountStatusEndpoint;
```

Comment:
this can be private

Reply:
we can't make endpoint private,how can we access it in other feature files then?
or else i need to create this field in each file